src/main/java/com/faforever/client/replay/ReplayServiceImpl.java

```diff
@@ -176,7 +177,7 @@ static String guessModByFileName(String fileName) {
                 replayInfos.add(new Replay(replayInfo, replayFile, featuredMod, mapBean.get()));
               });
         } catch (Exception e) {
-          logger.warn("Could not read replay file {} ({})", replayFile, e.getMessage());
+          logger.warn(MessageFormat.format("Could not read replay file {0}", replayFile), e);
```

Comment:
The logger supports using both, string replacement and exceptions as stack traces;

```
 logger.warn("Could not read replay file '{}'", replayFile, e);
```